libs/model/src/models/thread.ts

```diff
@@ -140,12 +140,52 @@ export default (
             transaction: options.transaction,
           });
 
+          const { topic_id, community_id } = thread.get({
+            plain: true,
+          });
+          const contestManagers = await sequelize.query<
+            z.infer<typeof ContestManager>
```

Comment:
should we make this query reusable?

Reply:
sure, just not sure what the directory structure would look like– don't want to just throw it in utils

Reply:
the current pattern for query interfaces is based on query files (.query.ts) ... you can try to place it in a new GetThreadContests.query.ts and call the body if you want to avoid the framework validation/auth... at least it will be isolated behind a reusable interface